src/elements/demo-image/DemoImageElementForm.tsx

```diff
@@ -24,14 +25,30 @@ export const ImageElementTestId = "ImageElement";
 export const ImageElementForm: React.FunctionComponent<Props> = ({
   fields,
   errors,
-  fieldViewSpecMap: fieldViewSpecs,
+  fieldViewSpecs,
 }) => (
   <div data-cy={ImageElementTestId}>
-    <FieldView fieldViewSpec={fieldViewSpecs.caption} />
-    <FieldView fieldViewSpec={fieldViewSpecs.altText} />
-    <FieldView fieldViewSpec={fieldViewSpecs.src} />
-    <FieldView fieldViewSpec={fieldViewSpecs.useSrc} />
-    <FieldView fieldViewSpec={fieldViewSpecs.optionDropdown} />
+    <Field
```

Comment:
This makes it supper easy to implement across the in task elements once we re base. Great re usable components